diff -Naur pgplot_old/sys_linux/f77_src/grsy00.f pgplot/sys_linux/f77_src/grsy00.f
--- pgplot_old/sys_linux/f77_src/grsy00.f	2019-10-10 22:02:12.040143813 +0900
+++ pgplot/sys_linux/f77_src/grsy00.f	2019-10-10 22:02:31.514221608 +0900
@@ -61,7 +61,7 @@
       INTEGER    FNTFIL, IER, INDEX(3000), NC1, NC2, NC3
       INTEGER    L, GRTRIM
       COMMON     /GRSYMB/ NC1, NC2, INDEX, BUFFER
-      CHARACTER*128 FF
+      CHARACTER*256 FF
 C
 C Read the font file. If an I/O error occurs, it is ignored; the
 C effect will be that all symbols will be undefined (treated as 
