diff -Naur pgplot_old/drivers/pndriv.c pgplot/drivers/pndriv.c
--- pgplot_old/drivers/pndriv.c	2019-10-08 20:19:03.179869351 +0900
+++ pgplot/drivers/pndriv.c	2019-10-08 20:19:31.632978681 +0900
@@ -222,7 +222,7 @@
 	return;
   }
 
-  if (setjmp(png_ptr->jmpbuf)) { /* not really sure what I'm doing here... */
+  if (setjmp(png_jmpbuf(png_ptr))) { /* not really sure what I'm doing here... */
 	fprintf(stderr,"%s: error in libpng while writing file %s, plotting disabled\n", png_ident, filename);
 	png_destroy_write_struct(&png_ptr,&info_ptr);
 	dev->error = true;
